CHANGES.md

```diff
@@ -24,6 +24,8 @@ Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos
 
 * [Metronome] Updates to fix daylight saving issues.
 
+* [Telegraf] Prometheus metrics can be collected from tasks in container networking mode. (DCOS-56018)
```

Comment:
@philipnrmn in hindsight I recognize that I don't understand this description very well.

I think we should have a more official name like "DC/OS Metrics: " as a prefix if at all. Doing this in an editorial cleanup.